org-code-javabuilder/lib/src/main/java/dev/javabuilder/WebSocketOutputAdapter.java

```diff
@@ -20,22 +20,26 @@ public WebSocketOutputAdapter(Session session) {
 
   @Override
   public void sendMessage(ClientMessage message) {
-    try {
-      endpoint.sendText(message.getFormattedMessage());
-    } catch (IOException e) {
-      e.printStackTrace();
-    } catch (IllegalStateException e) {
-      throw new InternalServerRuntimeException(InternalExceptionKey.CONNECTION_TERMINATED, e);
+    if (message.shouldAlwaysSend()) {
```

Comment:
Nit: feels more readable if we reversed the logic and could then just early return like `return if message.skipSend();`? Or at least shorten name to `shouldSend()`, what does the "always" represent?

Reply:
I had some struggles with naming on this one. The challenge is the specific neighborhood messages should only send when they hit `UserTestOutputAdapter`, and all other output adapters should ignore them. So `always` was supposed to signify every output adapter should send these. `skipSend` implies every output adapter should skip to me, which isn't quite what I intended, but if that makes more sense to you I can change it.

Reply:
> skipSend implies every output adapter should skip to me, which isn't quite what I intended, but if that makes more sense to you I can change it.

Every output adapter should skip sending these messages (except `UserTestOutputAdapter`), right? I'm suggesting set the default skipSend() to return `false`, and update the override method to return `true` where appropriate?

Reply:
so the issue is that the client message doesn't know which output adapter it is sending to, so it sends the default value every time. Then the output adapter decides to ignore that skip for some messages. Which is not a very clean pattern. I didn't want to have every output adapter need to parse every message if they did not need to.

Reply:
yep!